### PR TITLE
Model Layer Complete

### PR DESCRIPTION
**Pull request by Hammad887**
Added model (chatroom, message and user) and scripts
Wrote the scripts

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	@echo
+	@echo 'Commands:'
+	@echo '  build           		Compile project.'
+	@echo '  check           		Run linters.'
+	@echo '  format          		Format source code.'
+	@echo '  codegen         		Generate code.'
+	@echo '  test            		Run test case'
+	@echo '  prepare         		build dev container'
+	@echo '  clean       			remove dev temp folder'
+	@echo
```
Comment (shahzadhaider1) [inline Makefile:47]: please leave a blank line at the end of each file

File: models/chatroom.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package models
+
+type ChatRoom struct {
```
Comment (shahzadhaider1) [inline models/chatroom.go:3]: please add comments for Exported structs

**Review by shahzadhaider1 — CHANGES_REQUESTED** [review]
Good work. Some feedback

File: Makefile (hunk 1 above)
Comment (Hammad887) [reply Makefile:47]: Done

File: models/chatroom.go (hunk 2 above)
Comment (Hammad887) [reply models/chatroom.go:3]: Added

**Pull request merged**